app/services/return-requirements/check/returns-requirements.service.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+/**
+ * Orchestrates fetching and presenting the return requirements for the check page
+ * @module ReturnRequirementsService
+ */
+
+const ReturnRequirementsPresenter = require('../../../presenters/return-requirements/check/returns-requirements.presenter.js')
+const SessionModel = require('../../../models/session.model.js')
+const PurposeModel = require('../../../models/purpose.model.js')
```

Comment:
NIT - Alphabetical ordering

```suggestion
const PurposeModel = require('../../../models/purpose.model.js')
const ReturnRequirementsPresenter = require('../../../presenters/return-requirements/check/returns-requirements.presenter.js')
const SessionModel = require('../../../models/session.model.js')
```